docs/book/toc.md

```diff
@@ -44,6 +44,7 @@
 * [Practical MLOps](./advanced-guide/practical/practical-mlops.md)
   * [Stack Recipes](./advanced-guide/practical/stack-recipes.md)
   * [Switching Orchestration](./advanced-guide/practical/switching-orchestration.md)
+  * [Scheduling Pipeline Runs](./advanced-guide/practical/scheduling-runs.md)
```

Comment:
I would put this rather here: https://docs.zenml.io/advanced-guide/pipelines (In depth configuration)